ansible/roles-infra/infra-gcp-install-sdk/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- when: cloud_provider == "gcp"
+  environment:
+    PATH: '{{ output_dir }}/google-cloud-sdk/bin:/usr/bin:/usr/local/bin'
+    CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: "{{ gcp_credentials_file }}"
+    CLOUDSDK_COMPUTE_REGION: "{{ gcp_region }}"
+    CLOUDSDK_CONFIG: "{{ output_dir }}/.gcloud-{{ guid }}"
+    CLOUDSDK_CORE_PROJECT: "{{ gcp_project_id }}"
+  block:
+  - name: Remove previous Google Cloud SDK 
+    file:
+      path: "{{ output_dir }}/google-cloud-sdk"
+      state: absent
+    ignore_errors: yes
```

Comment:
why this `ignore_errors` is needed ?

Reply:
I don't think it is, I'll double check and remove it